include/tkey/led.h

```diff
@@ -17,6 +17,7 @@
 #define LED_WHITE (LED_RED | LED_GREEN | LED_BLUE)
 // clang-format on
 
+uint32_t get_led();
```

Comment:
Since we are only using 3-bits and it is fine to access and write with bytes. Should we consider making use of uint8_t instead? This would save us three bytes. Not much, but we are on the other hand a resource constrained device.

I would also argue to change to set_led(uint8_t led_value).

(I did a quick test, works fine).

Reply:
This would obviously be a API change, but I would also imagine that most calls so far are done with `set_led(LED_BLUE)`, so no changes are needed when updating.

Reply:
But doesn't work on qemu. In qemu only CDI and FW_RAM are byte readable. Needs further testing and probably fixing QEMU.

Reply:
Let's figure out the qemu parts first.